src/security/context.h

```diff
@@ -27,6 +24,8 @@ extern "C" {
 
 namespace datadog::nginx::security {
 
+static constexpr uintptr_t kBufferTag = 0xD47AD06;
```

Comment:
[suggestion] Can you add a comment to explain this magic number, its purpose and how it is used?

Reply:
it's an arbitrary tag to use for our buffers. modules usually use the value of the pointer to the module structure, but this has the advantage it's available at compile time. It's "datadog" written in hex digits